README.md

```diff
@@ -488,6 +488,17 @@ You can also apply different serialization `style` and `explode` as specified [h
 `@fastify/swagger` supports these options as shown in this example:
 
 ```js
+// Need to add a collectionFormat keyword to ajv in fastify instance
+const fastify = Fastify({
+  ajv: {
+    plugins: [
+      function (ajv) {
+        ajv.addKeyword({ keyword: 'collectionFormat' })
+      }
+    ]
```

Comment:
This should work too

```suggestion
    customOptions: {
      keywords: [ 'collectionFormat' ]
    }
```

Could you add a demo in the https://github.com/fastify/fastify-swagger/tree/master/examples folder?

Reply:
@Eomm I change plugins > customOptions and demo add examples folder :)